src/main/scala/org/constellation/p2p/PeerAPI.scala

```diff
@@ -49,7 +46,7 @@ class PeerAPI(override val ipManager: IPManager)(implicit system: ActorSystem, v
 
   implicit val serialization: Serialization.type = native.Serialization
 
-  implicit val executionContext: ExecutionContext = system.dispatchers.lookup("peer-api-dispatcher")
+  implicit val executionContext: ExecutionContext = dao.edgeExecutionContext // system.dispatchers.lookup("peer-api-dispatcher")
```

Comment:
Shouldn't this API get it's own pool? we don't need to use this dispatchers thing -- but we can just make one here.

Reply:
There was that strange behavior we observed earlier when the APIClient had a different pool from the spawner. I originally had all pools on edgeExecutionContext to rule out that issue, but this version I changed the signatureResponse and finishedCheckpoint to different ones, so it should be fine to change this back to it's own pool.